src/components/MultiSelect/FilterableMultiSelect.js

```diff
@@ -253,15 +257,46 @@ export default class FilterableMultiSelect extends React.Component {
       invalidText,
       translateWithId,
     } = this.props;
-    const className = cx(
-      `${prefix}--multi-select`,
-      `${prefix}--combo-box`,
-      containerClassName,
+    const inline = type === 'inline';
+    const wrapperClasses = cx(
+      `${prefix}--multi-select__wrapper`,
+      `${prefix}--list-box__wrapper`,
       {
-        [`${prefix}--list-box--light`]: light,
+        [`${prefix}--multi-select__wrapper--inline`]: inline,
+        [`${prefix}--list-box__wrapper--inline`]: inline,
+        [`${prefix}--multi-select__wrapper--inline--invalid`]:
+          inline && invalid,
+        [`${prefix}--list-box__wrapper--inline--invalid`]: inline && invalid,
       }
     );
-    return (
+    const className = ({ selectedItem }) =>
```

Comment:
`className` as a function is not seen in other components. Probably move this to render callback?

Reply:
this is how I implemented it for our [Dropdown](https://github.com/carbon-design-system/carbon-components-react/pull/2093/files#diff-700df727ae95bd5df9e9b8baa2eb6a1eR158) as well to get the dropdown state from downshift. Should I create a separate PR to change dropdown?

Reply:
Yes, it makes sense!